sw/device/lib/dif/dif_entropy.c

```diff
@@ -18,8 +18,9 @@
 static void set_config_register(const dif_entropy_t *entropy,
                                 const dif_entropy_config_t *config) {
   // TODO: Make this configurable at the API level.
-  uint32_t reg =
-      bitfield_bit32_write(0, ENTROPY_SRC_CONF_BOOT_BYPASS_DISABLE_BIT, 1);
+  uint32_t reg = 0;
+  // uint32_t reg =
+  //   bitfield_bit32_write(0, ENTROPY_SRC_CONF_BOOT_BYPASS_DISABLE_BIT, 1);
```

Comment:
Remove if unused. Does setting this bit cause any issues?

Reply:
yeah at the moment it does.  The fsm when I tested it locks up.  I'm checking in with Mark / Steve about it. 
I need to file an issue and add a TODO here.

Reply:
i've left a TODO there for this one.